scripts/release/issue-list.sh

```diff
@@ -55,7 +64,11 @@ function milestone_issues()
     if [[ "$line" == *"(task)"* ]]; then tasks="$tasks\n$line"; fi
   done <<< "$issue_list"
 
-  echo -e "$features\n$bugs\n$tasks"
+  features=$(discard_content_with_empty_issues "$features")
+  bugs=$(discard_content_with_empty_issues "$bugs")
+  tasks=$(discard_content_with_empty_issues "$tasks")
+
+  echo -e "$features$bugs$tasks"
```

Comment:
Also we can restructure the code. We can check each category and if it is not empty then we can directly print them to console them. If you want you can do that in a separate function .